docs/concepts/storage/persistent-volumes.md

```diff
@@ -70,6 +70,29 @@ Pods use claims as volumes. The cluster inspects the claim to find the bound vol
 
 Once a user has a claim and that claim is bound, the bound PV belongs to the user for as long as they need it. Users schedule Pods and access their claimed PVs by including a persistentVolumeClaim in their Pod's volumes block. [See below for syntax details](#claims-as-volumes).
 
+### Persistent Volume Claim Protection
+{% assign for_k8s_version="v1.9" %}{% include feature-state-alpha.md %}
+The purpose of the PVC protection is to ensure that PVCs in active use by a pod are not removed from the system as this may result in data loss.
+
+Note: PVC is in active use by a pod when the pod status is `Running`.
```

Comment:
What about when it has been scheduled to a node, but node hasn't finished bringing it to running?

Reply:
The tipping point is when a pod is assigned to a node.

I thought that the only thing a user sees are [pod phases](https://kubernetes.io/docs/concepts/workloads/pods/pod-lifecycle/#pod-phase) that do not have a separate phase for a pod that is not assigned to a node and for a pod that is assigned to a node. That's why I didn't include the `Pending` phase.

But it would be better to include the state when a pod is in `Pending` phase and is assigned to a node.